### PR TITLE
Add linter checkstyle

### PR DESCRIPTION
**Pull request by NDMyroshnychenko**
Add linter checkstyle

### CODE REVIEW

**Review by VL4DYSL4V — APPROVED** [review]
Linter configuration is compatible with specification

File: slidez-core/build.gradle [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+    id "io.spring.javaformat" version "0.0.27"
+}
+apply plugin: 'io.spring.javaformat'
+apply plugin: 'checkstyle'
+
+checkstyle {
+    toolVersion = "8.29"
+    configFile = rootProject.file('src/main/java/com/binarystudio/academy/slidez/infrastructure/checkstyle/checkstyle.xml')
```
Comment (klesogor) [inline slidez-core/build.gradle:12]: You shouldn't put linter files into the source code. Please, move it to java project root OR create a new folder `conofig` and place it there

**Review by klesogor — CHANGES_REQUESTED** [review]

File: slidez-core/build.gradle (hunk 1 above)
Comment (NDMyroshnychenko) [reply slidez-core/build.gradle]: I moved it, please, check

**Comment by klesogor** [comment]
I believe it was merged into develop as part of other PR. Should I close it?

**Review by Anatolii-Ivanov — APPROVED** [review]

**Comment by NDMyroshnychenko** [comment]
The branch is closed, because when making changes to the "50-editor-config" branch, the checkstyle was added

**Pull request closed without merging**